pkg/pool-manager/macentry.go

```diff
@@ -0,0 +1,37 @@
+package pool_manager
+
+import "time"
+
+func (m macEntry) hasPendingTransaction() bool {
+	return m.transactionTimestamp != nil
+}
+
+func (m macEntry) addTime(duration time.Duration) time.Time {
+	return m.transactionTimestamp.Add(duration)
+}
```

Comment:
Is not worth it since it's used just at `hasExpiredTransaction`

Reply:
you're right. Done